modules/twigextensionmodule/twigextension/TwigExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ *
+ * @link      https://deuxhuithuit.com
+ * @copyright Copyright (c) 2020 Deux Huit Huit
+ */
+
+namespace modules\twigextensionmodule\twigextension;
+
+use modules\twigextensionmodule\TwigExtensionModule;
+
+use Craft;
+
+use Twig_Extension;
```

Comment:
this is a good example if you have the same class name twice and you need to use both in the same file you can always "rename" one locally in the file with use.

e.g.: `use Twig_Extension as HelloWorld`

and Then create a class like so `class Twig_Extension extends HelloWorld` ...

Reply:
Good thinking! I did something similar in the module file.